content_schemas/dist/formats/call_for_evidence/frontend/schema.json

```diff
@@ -1020,12 +1020,6 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
-        "additional_topics": {
```

Comment:
I'll come back to this, or have you already checked? Ie do you have anything that confirms these fields (additional_topics and primary_topics) are specialist topics?

Reply:
I've grepped through email apps and found nothing. If you know that it is used somewhere else, please, let me know.

Reply:
Also, we have contract tests, but if they cannot be relied on, please, let me know.

Reply:
I'm a bit confused by https://github.com/alphagov/publishing-api/pull/2669/commits/ddaf127ef7da368238d376a16312dd0701c2b41f and https://github.com/alphagov/publishing-api/pull/2669/commits/513f78f0285b85b093b1747286b22b8bfe8eb64b

It looks like we need to remove topics, additional_topics and primary topic from the tags attribute of the email alert schema. And the relevant examples need to be updated. But I'm struggling to figure out if the examples you've added are including this schema. If they are, then it might be that there's some additional code changes needed which would form part of this change. 

So let's make this it's own PR. As yet I've not removed any code from email alert service or email alert api that might be using these fields. So hopefully the contract tests will fail in a helpful and informative way so we can link all the relevant work.